src/js/_enqueues/wp/media/models.js

```diff
@@ -237,8 +237,3 @@ media.query = function( props ) {
 		props: _.extend( _.defaults( props || {}, { orderby: 'date' } ), { query: true } )
 	});
 };
-
-// Clean up. Prevents mobile browsers caching.
-$(window).on('unload', function(){
-	window.wp = null;
-});
```

Comment:
Rationale for removal: https://core.trac.wordpress.org/ticket/55491#comment:17

However, if this depends on the server sending `Cache-Control: no-store`, then I'd like to further verify this. 

@spenserhale What happens if `no-store` is _not_ sent? Is there an issue then? I'd like to find a way to not have to send `no-store` in the admin so that we can benefit from bfcache, but if this depends on `no-store` then that would block enabling bfcache.

Reply:
+1 - fine to delete this super old bug fix/browser specific workaround. It is unlikely that the bug still exists, if it does we can always reintroduce a fix.

Reply:
Yep, same as above. Super old browser specific fix that seems okay to remove